### DISCUSSION TITLE
⼈⼯智慧、機器學習和深度學習 關係圖

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#37] [General]
![AI](https://user-images.githubusercontent.com/91121500/135222645-61f41c7c-5573-4e48-9530-6442d6a684ce.png)
⼈⼯智慧、機器學習和深度學習的關係圖如上：
機器學習是⼈⼯智慧的⼦領域之⼀ ; 深度學習也是機器學習的⼦領域之⼀ ; 深度學習
亦是⼈⼯智慧的⼦領域之⼀。

另外補充一下:⼈⼯智慧的發展可分成兩種：
a. 強⼈⼯智慧（真 AI）：我們的夢想是⼀個通俗的 AI，可以學各種⽂字、圖⽚，
、⾳樂......等等，⽬前技術尚未達成成熟階段。
b. 弱⼈⼯智慧（假 AI）：這是⽬前我們在討論的 AI，⼀個公式只能應⽤在⼀個領域
並且在這個領域上追過⼈類。

**Comment by EricAgumonJou** [comment]
AI 的技術可以讓機器更了解我們的思考，且能聽得⽐我們⼈類更清楚，看得也⽐我們遠，做得⽐我們更快更好，犯錯的機率也⽐我們少。
達成 AI 必須把⼈類的經驗學習起來，⽽我們模擬經驗的⽅式就是透過機器學習（統計），會在不同的情況下去利⽤不同的機器學習⽅式來找出公式。

**Comment by chia-kai-chang** [comment]
另外也有一些關係圖可以參考：

![image](https://user-images.githubusercontent.com/1768891/135251460-ca54b521-8b70-4996-b540-f520c87655aa.png)

![image](https://user-images.githubusercontent.com/1768891/135251574-e3d2604a-0195-4d93-9e54-f2ccfba9d7fe.png)


參考來源：https://chethankumargn.medium.com/artificial-intelligence-definition-types-examples-technologies-962ea75c7b9b

**Reply by enoch-lightning** [reply]
我看不太懂老師的這張人工智慧(artificial intelligence，AI)圖，怎麼上面的圖中，機器學習=機械學習(machine learning)左邊有深度學習(deep learning)的分支，下面的圖，只有機器學習而沒有深度學習。這樣機器學習與深度學習究竟是什麼關係？
😮‍💨  ❓ ❔

**Reply by EricAgumonJou** [reply]
你看我發的第一張圖就知道關係了，老師發的最後一張圖只是簡圖，其實綠色圈圈裡面還可以再畫深度學習的圈圈。

**Reply by EricAgumonJou** [reply]
你所說的 「機械」學習是？？？🤔🤔（應該沒有這個東西）

> 我看不太懂老師的這張人工智慧(artificial intelligence，AI)圖，怎麼上面的圖中，機器學習=機械學習(machine learning)左邊有深度學習(deep learning)的分支，下面的圖，只有機器學習而沒有深度學習。這樣機器學習與深度學習究竟是什麼關係？
> 😮‍💨 ❓ ❔

**Reply by enoch-lightning** [reply]
> 你看我發的第一張圖就知道關係了，老師發的最後一張圖只是簡圖，其實綠色圈圈裡面還可以再畫深度學習的圈圈。

收到。謝謝你。看起來「機器學習」的界線與「深度學習」的界線不太明顯。
